src/fsipy/automatedPostprocessing/postprocessing_h5py/spectrograms.py

```diff
@@ -168,11 +168,12 @@ def read_spectrogram_data(folder: Union[str, Path], mesh_path: Union[str, Path],
         start_t (float): Start time for data processing.
         end_t (float): End time for data processing.
         n_samples (int): Number of samples.
-        sampling_region (str): Region for sampling data ("sphere" or "domain").
+        sampling_region (str): Region for sampling data ("sphere" or "domain" or "box").
         fluid_sampling_domain_id (int): Domain ID for fluid sampling (used when sampling_region="domain").
         solid_sampling_domain_id (int): Domain ID for solid sampling (used when sampling_region="domain").
         fsi_region (list): x, y, and z coordinates of sphere center and radius of the sphere (used when
-            sampling_region="sphere").
+            sampling_region="sphere" ). In case of sampling_region="box", the list should contain [x_min, x_max,
```

Comment:
An extra space before the closing parenthesis in this line. Should be:
``` python
            sampling_region="sphere"). In case of sampling_region="box", the list should contain [x_min, x_max,
```